esx_service/utils/auth.py

```diff
@@ -326,6 +326,25 @@ def add_volume_to_volumes_table(tenant_uuid, datastore, vol_name, vol_size_in_MB
 
     return None
 
+def remove_volume_from_volumes_table(tenant_uuid, datastore, vol_name):
+    """ Remove volume from volumes table. """
+    _auth_mgr = get_auth_mgr()
+
+    logging.debug("remove volumes from volumes table(%s %s %s)", tenant_uuid, datastore,
```

Comment:
1. Should we specify the parameter name in the logging string to avoid confusion? Something like: "... (tenant=%s, datastore=%s, volume=%s)"
2. Nit: "remove volumes..." should be "remove volume..."
3. Should all log message start with capital letter?